.editorconfig

```diff
@@ -0,0 +1,4 @@
+[*]
```

Comment:
should this file be ignored?

Reply:
I meant for it to be included, it causes vim to use the local coding standards. I picked this up from other open source projects I contribute to (maybe consume is the right word here).